config/tests/host/io_vnic_scenario_goodpath.cfg

```diff
@@ -0,0 +1,34 @@
+avocado-misc-tests/io/net/virt-net/network_virtualization.py:NetworkVirtualization.test_add avocado-misc-tests/io/net/virt-net/network_virtualization.py.data/network_virtualization.yaml
+
```

Comment:
please remove this whit space and even all other places

Reply:
@PraveenPenguin this is to just differentiate a set of tests.. so the wrapper will not have any problem

Reply:
we want to keep the cfg that way.. and many other scenario cfgs are same

Reply:
we want to keep the cfg that way.. and many other scenario cfgs are same

Reply:
still i feel as it looks ugly

Reply:
@TasmiyaNalatwad anyway the wrapper ignoring the white lines,   this was added to differentiate each scenario in RQM,  like every set is a scenario separated by line.. if we club every thing it is difficult to know

Reply:
@abdhaleegit Sure abdul Will keep them as it is so that test scenarios are easily differentiated here. With this I am marking the review as resolved.
Thanks